mlf_core/create/template_creator.py

```diff
@@ -50,7 +51,10 @@ def process_common_operations(self, path: Path, skip_common_files=False, skip_fi
         """
         # create the common files and copy them into the templates directory (skip if flag is set)
         if not skip_common_files:
-            self.create_common_files()
+            self.create_common_files(domain='all', common_files_path=self.COMMON_FILES_PATH)
+            # if project is a mlflow project, copy all common files for mlflow projects
+            if self.creator_ctx.domain == 'mlflow':
```

Comment:
Would it be possibly to generalize this more with a switch statement?

Reply:
Yes, sure